bin/pygrb/pycbc_grb_trig_combiner

```diff
@@ -486,10 +453,14 @@ logging.info(
 # there should always be two segments in the offsource bins,
 # one before and one after the onsource.
 # Nevertheless, simply rely on there being at least one stretch of offsource.
-offsource_ntrials = [int(numpy.trunc(abs(b)/trialtime)) for b in bins["OFFSOURCE"]]
+offsource_ntrials = [int(numpy.trunc(abs(b)/trialtime))
+                     for b in bins["OFFSOURCE"]]
 
 # Seed random number generator
-numpy.random.seed(args.seed)
+if args.seed is None:
+    numpy.random.seed(args.seed)
+else:
+    numpy.random.seed(numpy.int64(args.seed))
```

Comment:
Can you explain this change? Why the explicit conversion to int64?

Reply:
It was taking the input as a string and failing.

Reply:
I/we kept it to the `None` default all along and it never failed, but when I actually used the option it complained that it was not an int64.

Reply:
A cleaner solution is to give the `--seed` argument a type of int. Then it will still be None if not given, and an int if given.

Reply:
Done!